src/app/components/resources/resources.component.ts

```diff
@@ -13,36 +18,100 @@ import { LanguageService } from '../../service/language.service';
 export class ResourcesComponent implements OnInit {
   resources: Resource[];
   languages: Language[];
+  resourceTypes: ResourceType[];
+  systems: System[];
 
   showInstructions = false;
+  showFilters = false;
   loadingResources = false;
   loadingLanguages = false;
   errorMessage: string;
+  filters: Filters;
+  totalFilters: number;
+  sortOrder: string;
 
   constructor(
     private resourceService: ResourceService,
     private languageService: LanguageService,
     private modalService: NgbModal,
+    protected systemService: SystemService,
+    protected resourceTypeService: ResourceTypeService,
   ) {}
 
   ngOnInit(): void {
     this.loadResources();
     this.loadLanguages();
+    this.loadFilters();
   }
 
-  loadResources(): void {
-    this.loadingResources = true;
+  blankFilters(): Filters {
```

Comment:
Since this function returns empty filters and doesn't change or reset the current filters, I think the current name is appropriate. Although it could be called `getBlankFilters`. Thoughts?

Reply:
I like `getBlankFilters`